### PR TITLE
Update/fetch orderform

### PR DESCRIPTION
**Pull request by ClaudivanFilho**
#### What is the purpose of this pull request?
- Add request to get the orderFormId

https://orderform--storecomponents.myvtex.com/shelf

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/ShelfContent.js [hunk 1]
```diff
@@ -72,10 +75,11 @@ class ShelfContent extends Component {
   }
 
   slideFallback = (item, key) => {
-    const { summary } = this.props
+    const { summary, data: { orderForm } } = this.props
+    const orderFormId = orderForm && orderForm.orderFormId
     return (
       <div key={key} className={`${VTEXClasses.SLIDE_CLASS} pa4`}>
-        <ShelfItem extensionId="shelfitem" item={item} summary={summary} />
+        <ShelfItem extensionId="shelfitem" item={item} summary={summary} orderFormId={orderFormId} />
```
Comment (vitoriaHeliane) [inline react/ShelfContent.js]: ExtensionId not more necessary.

File: react/graphql/orderFormQuery.gql [hunk 2]
```diff
@@ -0,0 +1,16 @@
+query orderForm @context(scope: "private") {
```
Comment (vitoriaHeliane) [inline react/graphql/orderFormQuery.gql]: With you just need the orderFormId, I think you could remove the others fields.

File: react/ShelfContent.js [hunk 3]
```diff
@@ -124,4 +133,11 @@ ShelfContent.propTypes = {
...
 }
 
-export default ShelfContent
+const options = {
+  options: () => ({
+    ssr: false,
+  }),
+}
+const ShelfContentWithQuery = graphql(orderFormQuery, options)(ShelfContent)
+ShelfContentWithQuery.propTypes = ShelfContent.propTypes
```
Comment (lucasecdb) [inline react/ShelfContent.js]: why you need this?

File: react/Shelf.js [hunk 4]
```diff
@@ -9,7 +9,7 @@ import ScrollTypes, { getScrollNames, getScrollValues } from './ScrollTypes'
 import OrdenationTypes, { getOrdenationNames, getOrdenationValues } from './OrdenationTypes'
 import VTEXClasses from './CustomClasses'
 
-import productsQuery from './queries/productsQuery.gql'
+import productsQuery from './graphql/productsQuery.gql'
```
Comment (brunojdo) [inline react/Shelf.js]: we use queries and mutations folders.

File: react/ShelfContent.js (hunk 1 above)
Comment (brunojdo) [inline react/ShelfContent.js]: I don't think this is needed.

**Comment by ClaudivanFilho** [comment]
This feature is not necessary

**Pull request closed without merging**